webui/react/src/components/Metadata.tsx

```diff
@@ -51,6 +50,11 @@ const Metadata: React.FC<Props> = ({ trial }: Props) => {
 
   const treeData = (trial?.metadata && getNodes(trial?.metadata)) ?? [];
 
+  if (treeData.length === 0) {
```

Comment:
can we move the check for metadata up to the trial details overview?

Reply:
fixed